ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -232,8 +232,8 @@ tests:
     - chain: openshift-qe-node-density
     workflow: openshift-qe-installer-aws-ovn-fips
   timeout: 5h0m0s
-- always_run: false
-  as: node-density-24nodes-ovn-fips
+- as: node-density-24nodes-ovn-fips
+  cron: 0 8 9,23 * *
```

Comment:
@afcollins Same reason as above. Pls move this to always_run: false.

Reply:
done